wrapanapi/systems/rhevm.py

```diff
@@ -855,23 +862,31 @@ def list_cluster(self, **kwargs):
     def get_disks(self, name):
         return self.api.system_service().disks_service().list(search=name)
 
-    def list_disks(self, status=None, **kwargs):
+    def list_disks(self, status=None, sparse=None, disk_id=None, **kwargs):
         """
         List all of the disks present on RHV
 
         Keywords:
             status (optional) -- status of the disk.One of OK, LOCKED, ILLEGAL.
+            sparse (optional) -- disk allocation. Sparse is a thin disk.
 
         Returns:
-            list of disk names(str)
+            list of disk names(str) or disk ids(str) if disk_id=True
         """
         disks_list = self.api.system_service().disks_service().list()
+        if sparse is not None and status is None:
```

Comment:
would it be appropriate to write:
`if sparse and status is None:` ?

Reply:
yes, thanks